enterprise/models.py

```diff
@@ -169,6 +178,28 @@ def requests_data_sharing_consent(self):
         """
         return self.enable_data_sharing_consent and self.enforce_data_sharing_consent != self.EXTERNALLY_MANAGED
 
+    def get_course_enrollment_url(self, course_run_key):
+        """
+        Return enterprise landing page url for the given course.
+
+        Arguments:
+            course_run_key (str): The course run id for the course to be displayed.
+        Returns:
+            (str): Enterprise landing page url.
+        """
+        if configuration_helpers is None:
```

Comment:
The more we use this pattern (which I created when moving consent stuff over initially), the less happy I am with it. We're going to have to deal with it eventually. Not here, but eventually.